flepimop/gempyor_pkg/src/gempyor/inference.py

```diff
@@ -24,6 +45,7 @@
 from . import seir, model_info
 from . import outcomes, file_paths
 from .utils import config, Timer, read_df, as_list
+from typing import Literal
```

Comment:
Up to you if you want to do this now or later, but let's try to follow PEP8 for imports going forward (i.e. move this up to stdlib imports rather than with local pkg imports): https://peps.python.org/pep-0008/#imports